src/Tribe/Views/V2/Rest_Endpoint.php

```diff
@@ -188,13 +209,14 @@ public function enable_ajax_fallback() {
 	 * the REST API still have the Views V2 working.
 	 *
 	 * @since  4.9.7
+	 * @since TBD Changed the mock request HTTP method to POST (was GET).
 	 *
 	 * @param  array $params Associative array with the params that will be used on this mocked request
 	 *
 	 * @return Request The mocked request.
 	 */
 	public function get_mocked_rest_request( array $params ) {
-		$request = new Request( 'GET', static::ROOT_NAMESPACE . '/html' );
+		$request = new Request( 'POST', static::ROOT_NAMESPACE . '/html' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: 1782e1487691599d5deaab835a6ec696</sub>